java/datastructures/lib/src/test/java/datastructures/businesstrip/BusinessTripGraphTest.java

```diff
@@ -0,0 +1,43 @@
+package datastructures.businesstrip;
+
+import static org.junit.Assert.*;
+import org.junit.Test;
+import datastructures.graph.Graph;
+import datastructures.graph.Vertex;
+
+public class BusinessTripGraphTest {
+  @Test
+  public void businessTripTest() {
+    Graph<String> graph = new Graph<>();
```

Comment:
Take a look at your Graph constructors. Your empty constructor (the one you're calling here) doesn't create an `adjacencyLists`. When you pass a graph with a null `adjacencyLists` property to your `businessTrip` method, you'll get an error. To fix this, you could pass a value in here (`Graph<String> graph = new Graph<>(5)`, for example), or you could add an `adjacencyLists` to the graph you create here.